plugins/woocommerce/tests/php/src/Internal/PushNotifications/DataStores/PushTokensDataStoreTest.php

```
@@ -138,29 +139,29 @@ public function test_it_can_delete_push_token() {
 
 	/**
-	 * @testdox Tests the read method throws exception when push token has no
-	 * ID.
+	 * @testdox Tests the read method throws exception when push token ID is
+	 * invalid.
 	 */
-	public function test_it_throws_exception_when_reading_push_token_without_id() {
+	public function test_it_throws_exception_when_reading_push_token_with_invalid_id() {
 		$data_store = new PushTokensDataStore();
-
 		$push_token = new PushToken();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$push_token'. (undefined)

(UnusedLocalVariable)